packages/repository-json-schema/src/build-schema.ts

```diff
@@ -58,19 +58,24 @@ export function getJsonSchema(ctor: Function): JsonDefinition {
  */
 export function modelToJsonSchema(ctor: Function): JsonDefinition {
   const meta: ModelDefinition = ModelMetadataHelper.getModelMetadata(ctor);
-  const schema: JsonDefinition = {};
+  const result: JsonDefinition = {};
 
-  const isComplexType = (constructor: Function) =>
-    !includes([String, Number, Boolean, Object], constructor);
+  const defineSchemaProperty = (
+    schema: JsonDefinition,
+    prop: string,
+    propCtor: Function | string,
+  ) => {
```

Comment:
Please extract `defineSchemaProperty` to a regular top-level function. Right now, the diff contains too many whitespace changes which makes it impractical to review. Also `modelToJsonSchema` is becoming too long in the terms of lines of code, we should start refactoring bits of code into standalone functions defined outside of `modelToJsonSchema`.